src/auction.ts

```diff
@@ -548,6 +548,7 @@ export function auctionCallbacks(auctionDone, auctionInstance, {index = auctionM
 
     bidderRequest.bids.forEach(bid => {
       if (!bidResponseMap[bid.bidId]) {
+        addCommonResponseProperties(bid, bid.adUnitCode);
```

Comment:
this will add cpm = 0 and a couple other properties that don't seem to make sense. Can you scope this down to the timing metrics you desire?